### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 5

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#455] [Esami & Esoneri M-Z]
## Traccia

<img width="632" alt="image" src="https://github.com/user-attachments/assets/c7b74c37-1cf6-4c1d-ac70-cf5093e36e73">

**Comment by rimaout** [comment]
<img width="800" alt="image" src="https://github.com/user-attachments/assets/d37e224a-aa6a-49eb-a7a6-a463942d39c8">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/503d83e2-9858-402a-bd5b-3d920c5d4e53)
